crates/orchestrator/tests/common/constants.rs

```diff
@@ -0,0 +1,3 @@
+pub const MADARA_RPC_URL: &str = "http://localhost:9944";
+pub const MONGODB_CONNECTION_STRING: &str = "mongodb+srv://admin:Km6MuqgzRcRpSH2@cluster0-1.hhmuw56.mongodb.net/";
```

Comment:
you might want to revoke keys if you committed this by mistake :)

Reply:
This is a free mongo cluster. I thought it'd be simpler to store it here, plus I can just close it down if it's overused 😅 .
Feel free to use this for tests, as it has already set up the DBs ig